identityprovider/httpprovider.go

```diff
@@ -7,20 +7,22 @@ import (
 	"github.com/iden3/go-iden3-core/core"
 	"github.com/iden3/go-iden3-core/keystore"
 	"github.com/iden3/go-iden3-core/merkletree"
+	"github.com/iden3/go-iden3-core/services/claimsrv"
+	"github.com/iden3/go-iden3-core/services/identityagentsrv"
 	"github.com/iden3/go-iden3-crypto/babyjub"
 	"gopkg.in/go-playground/validator.v9"
 )
 
 type KeyStorer interface {
-	SignBaby(pk *babyjub.PublicKeyComp, msg []byte) (*babyjub.SignatureComp, error)
+	SignBaby(pk *babyjub.PublicKeyComp, prefix keystore.PrefixType, msg []byte) (*babyjub.SignatureComp, int64, error)
```

Comment:
TODO: Leave the interface without the prefix nor date to maintain it minimal.  Use a wrapper that uses the interface to apply the prefix and date